platforms/spring-boot/components-starter/pom.xml

```diff
@@ -286,6 +287,7 @@
     <module>camel-spring-boot-starter</module>
     <module>camel-spring-cloud-netflix-starter</module>
     <module>camel-spring-cloud-starter</module>
+    <module>camel-spring-dm-starter</module>
```

Comment:
I guess this one is not needed, maybe you still have the folder in your repo.

Reply:
Yeah he didnt run git clean -d -f, anyway if you regen the project it will go away

Reply:
@davsclaus  what do you mean by regen the project, is this a speciel maven cmd?